coreapi/request.go

```diff
@@ -244,3 +248,14 @@ type Schedule struct {
 	// start_time will be current time and first payment will happen on one interval after current time.
 	StartTime string `json:"start_time,omitempty"`
 }
+
+type PaymentAccount struct {
```

Comment:
Should be `PaymentAccountReq` to maintain naming consistency.

Reply:
done